WebDriverAgent.xcodeproj/xcshareddata/xcschemes/IntegrationApp.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0900"
+   LastUpgradeVersion = "1000"
```

Comment:
I this necessary to update this file?

Reply:
It's not necessary. As I remember, I just update repo to "recommended settings". If I revert its update, it will cause merge conflict in further pr's. So, if it is not a serious problem, I will leave this file as it is. It would would be updated in next pr's.